Library/Homebrew/test/test_dependency_collector.rb

```diff
@@ -39,7 +39,8 @@ def test_dependency_tags
 
   def test_requirement_creation
     @d.add :x11
-    assert_instance_of X11Requirement, find_requirement(X11Requirement)
+    req = OS.mac? ? X11Requirement : XorgRequirement
+    assert_instance_of req, find_requirement(req)
```

Comment:
I don't think so. For this test in particular, the purpose seems to be to confirm that delegation from the `:x11` symbol to X11Requirement is working. For that reason, I changed it such that it delegates to the correct class depending on OS.